modules/lifestreetBidAdapter.js

```diff
@@ -0,0 +1,146 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'lifestreet';
+const ADAPTER_VERSION = 'prebidJS-3.0';
```

Comment:
This is just a suggestion, you can use something like this:

```
version = '$prebid.version$'
```
to get the exact version of pbjs, but if 3.0 is precise-enough, ok :)